### PR TITLE
hpctools: 3.5.1 → 3.5.2

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

File: var/spack/repos/builtin/packages/hpctools/package.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ class Hpctools(CMakePackage):
     url      = "ssh://bbpcode.epfl.ch/hpc/HPCTools"
 
     version('develop', git=url)
-    version('3.5.1', tag='3.5.1', git=url, preferred=True)
+    version('3.5.2', tag='v3.5.2', git=url, preferred=True)
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/hpctools/package.py:35]: `v` : changed to new tag convention? :)

**Review by pramodk — APPROVED** [review]

File: var/spack/repos/builtin/packages/hpctools/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/hpctools/package.py:35]: Yes, I guess. I use `v` elsewhere, too.

**Comment by matz-e** [comment]
Please retest

File: var/spack/repos/builtin/packages/hpctools/package.py (hunk 1 above)
Comment (ferdonline) [reply var/spack/repos/builtin/packages/hpctools/package.py:35]: v is being deprecated in some open source projects... anyway, I guess every project always ends up having both :p  
But for the sake of consistency, could we all agree in a convention?
PS: I normally don't use `v`

Comment (matz-e) [reply var/spack/repos/builtin/packages/hpctools/package.py:35]: I would prefer `v`? :P

Comment (pramodk) [reply var/spack/repos/builtin/packages/hpctools/package.py:35]: In my opinion, project should use the style that they are currently using (with or without v, there are no hard rule about this)

**Pull request merged**